Makefile.toml

```diff
@@ -69,9 +69,7 @@ category = "Check"
 command = "${CARGO}"
 args = [
     "clippy",
-    "--no-default-features",
-    "--features",
-    "contract",
+    "--all-features",
```

Comment:
This feels a little disingenuous. In a typical Rust project where you have some code that is only compiled when there is a feature present, and none of those features are mutually exclusive, then this works fine. But in our case we have code that is compiled on the presence _or absence_ of a feature (namely `contract`). This means `--all-features` does not actually check all the code we care about because it will miss what is behind `#[cfg(not(feature = "contract"))]`.

That said, the original `--features contract` version of this command obviously didn't check that code either, but my main point is `--all-features` might give a false sense of security. Maybe it makes sense to add multiple `clippy` commands with different feature sets.

Reply:
I had been thinking about this as well as I had it miss a problem just running it normally.